README.md

```diff
@@ -2,11 +2,77 @@
   <img alt="Nimble logo" src="https://assets.nimblehq.co/logo/light/logo-light-text-320.png" />
 </p>
 
-<p align="center">
-  <strong>Our Github Action publishes your documentation to your project's Github Wiki!</strong>
-</p>
+<h3 align="center">Publish Github Wiki Action</h3>
+<p align="center">This action helps syncs your project's documentation with your project's Github Wiki.</p>
+
+-------
+
+## Usage
+
+Add the following workflow as a step in your existing workflow or a new one, replacing `USER_NAME` and `USER_EMAIL` with your account/bot information.
+
+```yml
+- name: Publish Github Wiki
+  uses: nimblehq/publish-github-wiki-action@v1.0
+  env:
+    USER_NAME: John Smith
+    USER_EMAIL: john_smith@email.com
+    USER_ACCESS_TOKEN: ${{ secrets.USER_ACCESS_TOKEN }}
+```
+
+<!-- TODO: Allow users to customize the wiki directory folder -->
+This workflow will synchronize the `.github/wiki` folder with your project's Github Wiki.
+
+## Setup
+
+To use this workflow, your project must have the following:
+
+- The Github Wiki must be enabled and initialized.
+- A valid Personal Access Token must be provided to the workflow.
+
+Below are steps to ensure proper setup:
+
+1. **Enable the Wiki feature**
+
+   You can enable the Wiki through your project's "Settings" tab.
+
+   ![Wiki Feature in Settings](https://user-images.githubusercontent.com/34730459/162665798-a6464c67-c74a-427e-9221-4df8310fe9cb.png)
+
+2. **Create at least one Wiki Page in the "Wiki" tab**
+
+   This is required to initalize the project's Wiki repository.
```

Comment:
```suggestion
   This is required to initialize the project's Wiki repository.
```

Reply:
Thanks! Updated in 11592dcf0592b56bf65f197077eb0ef1d8f206b3